app/Http/Controllers/Api/UsersController.php

```diff
@@ -187,6 +188,10 @@ public function index(Request $request)
             $users->has('accessories', '=', $request->input('accessories_count'));
         }
 
+        if ($request->filled('autoassign_licenses')) {
+            $users->has('autoassign_licenses', '=', $request->input('autoassign_licenses'));
```

Comment:
I'm not sure about this one - the `has()` relationship for accessories, above, is about a _relationship_ so the `has()` is appropriate. Here, I think you just want `->where()`?